llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2566,6 +2623,11 @@ struct VPWidenLoadEVLRecipe final : public VPWidenMemoryRecipe, public VPValue {
     setMask(Mask);
   }
 
+  VPWidenLoadEVLRecipe *clone() override {
+    llvm_unreachable("VPWidenLoadEVLRecipe cannot be cloned");
```

Comment:
This and `VPWidenStoreEVLRecipe *clone() override` are unrelated to the patch IIUC, would be good to split off